### DISCUSSION TITLE
myMPD spotted in commercial product?

### DISCUSSION THREAD

**Discussion by GioF71** [jcorporation/myMPD#1140] [General]
Hello, please have a look at [this video](https://www.youtube.com/watch?v=_FDcCwVOxZc), at around 20:04, from what I understand, that looks very much like myMPD. 
But I do not see myMPD mentioned on the [product page](https://ideonaudio.com/eos-stream/).
At least mpd is mentioned, but nothing about mympd.
Did you know about this? 

Cheers
Giovanni

**Comment by GioF71** [comment]
Also in the [manual](https://ideonaudio.com/wp-content/uploads/2023/05/eos-Stream_2023-manual.pdf)... no mentions about mpd and/or myMPD.

**Comment by jcorporation** [comment]
Thanks for the hint. It is an old version myMPD, the repo is here: https://github.com/ideonaudio/myMPD

I do not know the GPL exactly, it is required that they must mention myMPD?

**Comment by GioF71** [comment]
I am not a licensing expert... but mentioning at least, regardless of the obligations, I believe it would be a honorable action in any case...

**Reply by jcorporation** [reply]
Edit, this should be the correct product: https://www.cm-audio.net/produkt/ideon-audio-streamer-eos-stream/

It is not cheap...

**Reply by GioF71** [reply]
Not cheap at all, especially if you see the internals. There is a modified mini-itx mainboard (does look also a little dated). For sure there is a fancy power supply and a "reclocker" module.
Maybe those modification might improve the sound, although there are no measurements. 
All the rest looks like a collection of open-source and/or free software (roon bridge is not open source iirc), so I believe the markup is huge.

**Comment by jcorporation** [comment]
Notified the MPD project: https://github.com/MusicPlayerDaemon/website/pull/93

**Comment by GioF71** [comment]
They also mention wrong things in the manual, look at this:

```
Enable DoP: (enabled/disabled):
should be enabled for DACs that
do not support the transcode
for DSD playback. It basically
converts everything into PCM
format.
```

While, from the MPD documentation, dop is...

`DoP (DSD over PCM) playback. This wraps DSD inside fake 24 bit PCM according to the DoP standard. Requires a DAC that supports DSD. No support from ALSA and the sound chip required (except for bit-perfect 24 bit PCM support).`

**Comment by ideonaudio** [comment]
Hello from IdeonAudio support Team.

Please take another look to our pages for the two streamers we have. Absolute Stream and EOS Stream.

We have the Source code we use for our version gui and we had a cooperation with the MPD developer(Max), in the recent past.
Actually two new features in mpd code, was after our cooperation with Max!

EOS Stream is our new product and it is my mistake that i didn't update the page sooner. Sorry for that :(

Thank you @GioF71 for spotted this error to our manual. We will fix it ASAP.

JC congrats for your wonderful project. I personally keep following the progress you make...

PS. May JC do not remember it but before we begin to customize the myMPD to our needs, we send an email asking if he wants to make the customization we needed that time :) 

PS2. Of course we do not sell any of the open source software we use to our clients.

**Reply by GioF71** [reply]
Personally I am very happy to hear this from the company. Not everyone does that.

**Reply by jcorporation** [reply]
It is all fine for me, but I am curious. Which two new features are added to mpd through your cooperation?

I remember this customization request. It was at an very early stage of myMPD, was ideon named collybia before?

**Reply by ideonaudio** [reply]
Let me answer to your curiosity :)

The first one was added to ver 0.23.3
  - alsa: add option "stop_dsd_silence" to work around DSD DAC noise

and the second to ver 0.23.4
  - alsa: add option "thesycon_dsd_workaround" to work around device bug

You are right. I was under Collybia project (i still am but i don't have time to begin again for now). Actually Collybia project was the reason to meet IdeonAudio guys and start the collaboration for this wonderful project.

PS. The propose is always open JC! :)
